pymemcache/client/retrying.py

```diff
@@ -0,0 +1,48 @@
+import collections
+import functools
+import inspect
+import socket
+import time
+import logging
+
+import pymemcache.exceptions as pymemexception
+
+logger = logging.getLogger(__name__)
+
+def retry(func):
+    @functools.wraps(func)
+    def func_call(*args, **kwargs):
+        ntries=15
+        while ntries:
+            try:
+                return func(*args, **kwargs)
+            except pymemexception.MemcacheUnexpectedCloseError as e:
```

Comment:
Same thing here, I think that we need to provide strategies. Either we allow handle all the kinds of exceptions, or only the pymemcache exceptions, or again only one specific type of exception passed by user.

For now it's just a dumb PoC for my own usage.